### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by kay0u**
## Solution
- *Update nginx conf*
- *Use the new permission system*
- *[13.8.0](https://about.gitlab.com/releases/2021/01/22/gitlab-13-8-released/)*
- *[13.8.1](https://about.gitlab.com/releases/2021/01/26/gitlab-13-8-1-released/)*
- *[13.8.2](https://about.gitlab.com/releases/2021/02/01/security-release-gitlab-13-8-2-released/)*
- *[13.8.3](https://about.gitlab.com/releases/2021/02/05/gitlab-13-8-3-released/)*
- *[13.8.4](https://about.gitlab.com/releases/2021/02/11/security-release-gitlab-13-8-4-released/)*

## PR Status
- [ ] Code finished.
- [ ] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [ ] Can be reviewed and tested.

## Validation
---
- [ ] **Code review** : 
- [ ] **Approval (LGTM)** :  
*Code review and approval have to be from a member of @YunoHost-Apps/apps-group*
- **CI succeeded** : 
[Build Status](https://ci-apps-hq.yunohost.org/jenkins/job/gitlab_ynh%20PR-NUM-/)  
*Please replace '-NUM-' in this link by the PR number.*  
When the PR is marked as ready to merge, you have to wait for 3 days before really merging it.

### CODE REVIEW

**Comment by kay0u** [comment]
!testme

**Comment by kay0u** [comment]
By the power of systemd, I invoke The Great App CI to test this Pull Request!

File: scripts/install [hunk 1]
```diff
@@ -243,8 +242,7 @@ ynh_script_progression --message="Configuring SSOwat..." --weight=1
 
 # Make app public if necessary
 if [ $is_public -eq 1 ]; then
-	# unprotected_uris allows SSO credentials to be passed anyway.
-	ynh_app_setting_set --app=$app --key=unprotected_uris --value="/"
+	ynh_permission_update --permission="main" --add "visitors"
```
Comment (kay0u) [inline scripts/install:245]:
```suggestion
	ynh_permission_update --permission="main" --add="visitors"
```

**Comment by kay0u** [comment]
!testme

**Comment by Christophe31** [comment]
version bump 13.8.4 + may be interesting security wise. <https://www.cert.ssi.gouv.fr/avis/CERTFR-2021-AVI-116/>

**Review by yalh76 — DISMISSED** [review]
Quick review, LGTM

**Pull request merged**